Example/Example/AppFlowController.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+import FlowControllerKit
+
+class AppFlowController {
+
+    let storyboard = UIStoryboard(name: "Main", bundle: nil)
+
+    var valueVC: ValueViewController!
```

Comment:
I prefer printing out the full name, especially since we're doing it on the line below. 

It also seems like this can me a `let`

Reply:
Does it have to be public?

Reply:
It is not public, default in Swift is internal. but yes, private is preferred.